ui/library/src/components/PageWrapper.tsx

```diff
@@ -33,29 +34,32 @@ export const PageWrapper: React.FunctionComponent<Props> = ({
   ...extraProps
 }: Props) => {
   const { rotation, scale } = React.useContext(TransformContext);
-  const { pageDimensions } = React.useContext(DocumentContext);
+  const { pageDimensions, getOutlineTargets } = React.useContext(DocumentContext);
 
   // Don't display until we have page size data
   // TODO: Handle this nicer so we display either the loading or error treatment
   if (!pageDimensions) {
     return null;
   }
 
+  const getPageStyle = React.useCallback(() => {
+    const styles: Record<string, unknown> = computePageStyle(pageDimensions, rotation, scale);
+    return styles;
+  }, [pageDimensions, rotation, scale]);
+
   const getWidth = React.useCallback(() => {
     return getPageWidth(pageDimensions, rotation);
   }, [pageDimensions, rotation]);
 
-  const getPageStyle = React.useCallback(() => {
-    return computePageStyle(pageDimensions, rotation, scale);
-  }, [pageDimensions, rotation, scale]);
+  const outlineTargets = getOutlineTargets({ pageIndex, scale, rotation, pageDimensions });
 
   // Width needs to be set to prevent the outermost Page div from extending to fit the parent,
   // and mis-aligning the text layer.
   // TODO: Can we CSS this to auto-shrink?
   return (
     <div
       id={generatePageIdFromIndex(pageIndex)}
-      className="reader__page"
+      className={classnames('reader__page')}
```

Comment:
I think you don't have to use classnames here

Reply:
@yensung alright let me remove it

Reply:
@yensung ✅